src/OSPSuite.UI/Views/SimulationOutputMappingView.cs

```diff
@@ -43,6 +45,15 @@ public SimulationOutputMappingView()
       public void AttachPresenter(ISimulationOutputMappingPresenter presenter)
       {
          _presenter = presenter;
+         gridView.OnDeleteSelectedRows = selectedItems =>
+         {
+            var selectedDTOs = selectedItems
+               .OfType<SimulationOutputMappingDTO>()
+               .ToList();
```

Comment:
After you move the context menu into this file you can get the list of DTO's without checking type

`var selectedDTOs = gridView.GetSelectedRows().SelectMany(x => _gridViewBinder.SelectedItems(x));`

The binder will always return the DTO